src/index.ts

```diff
@@ -15,6 +15,16 @@ import evaluateXPathToNumbers from './evaluateXPathToNumbers';
 import evaluateXPathToString from './evaluateXPathToString';
 import evaluateXPathToStrings from './evaluateXPathToStrings';
 import executePendingUpdateList from './executePendingUpdateList';
+import castToType from './expressions/dataTypes/castToType';
+import createAtomicValue from './expressions/dataTypes/createAtomicValue';
+import {
```

Comment:
I would not expect _any_ changes in this file at all

Reply:
Since some of the tests also use ValueType, EllipsisType, ... Can they stay in the export section or should all code that gets called by the tests get converted to strings?

Reply:
Tests that use that type can just import them directly from the package. You'll see only unit tests in the `expressions` folder import it, not the integration tests in the `parsing` folder